lib/executables/validate.js

```diff
@@ -0,0 +1,146 @@
+/* eslint-disable no-use-before-define */
+const assertion = require('assert');
+const fs = require('fs');
+const path = require('path');
+const utils = require('../helpers/utils');
+const { print } = require('../helpers/log');
+
+const VIEW_CLASSES = [
+  'TextFieldView', 'TextAreaView', 'CheckBoxView', 'SelectView', 'MultiSelectView', 'PasswordFieldView', 'OAuthFieldView',
+];
+
+const schemas = {};
+let valid = true;
+
+const assert = {
+  info: (msg) => { print.info(`\t✅   ${msg}`); },
+  warn: (val, msg) => {
+    try {
+      assertion(val, msg);
+      return true;
```

Comment:
are these return values used at all? same in the catch and also for `error`

Reply:
Yes they are used throughout